pages.it/common/git-checkout.md

```diff
@@ -0,0 +1,32 @@
+# git checkout
+
+> Cambia rami o ripristina i file dell'albero di lavoro.
+> Maggiori informazioni: <https://git-scm.com/docs/git-checkout>.
+
+- Crea e passa ad un nuovo ramo:
+
+`git checkout -b {{nome_del_ramo}}`
```

Comment:
```suggestion
`git checkout -b {{nome_ramo}}`
```
I'd change `nome_del_ramo` to `nome_ramo`, which is simpler but still correct.

NOTE: Only writing this one time, but it applies to all the occurrences in the page.